pkgs/desktops/xfce/applications/orage.nix

```diff
@@ -13,8 +13,17 @@ stdenv.mkDerivation rec {
     sha256 = "0qlhvnl2m33vfxqlbkic2nmfpwyd4mq230jzhs48cg78392amy9w";
   };
 
+  patches = [
+    # Fix build with libical 3.0
+    (fetchpatch {
+      name = "fix-libical3.patch";
+      url = https://git.archlinux.org/svntogit/packages.git/plain/trunk/libical3.patch?h=packages/orage&id=7b1b06c42dda034d538977b9f3550b28e370057f;
```

Comment:
it is 404

Reply:
Maybe switch to https://salsa.debian.org/xfce-team/apps/orage/-/raw/265e058a2656eb64beb98d4509dd1d3a1e9992e7/debian/patches/02_libical3.patch